docs: homepage, concepts cleanup (#2643)
caos/zitadel: docs/docs/guides/solution-scenarios/b2c.mdx

---
title: B2C
---

import Column from "../../../src/components/column";

## Business to Consumer

Users in general come with different needs. You may have end users, employees, or even customers from other parties (B2B).
This groups of users usually don't share the same intentions and use applications differently.
When planning your applications, investing time in researching your apps architecture is vital for later feature upgrades and enhancements as those changes come in with heftier price points if you have to make bigger changes.

This guide introduces you to the grouping and structuring of ZITADEL projects which forms the base for all projects. This can be used as a quick start to the [B2B scenario](./b2b), which is merely focused on planning considerations if you are having projects with multiple organizations.

The journey of this guide starts with creating an Organization, the outermost layer of ZITADEL, as it is the vessel for projects, roles, applications and users.
Creation can be done from [ZITADEL Console](https://console.zitadel.ch/org/create). You can choose your current account for the organization owner or create a new one.

> Depending on your Software Development Life Cycle (SDLC) you will have to create multiple organizations to keep your data from different environments separate.
> 
> | Environment | Organization Name | Description                                       |
> | :---------- | :---------------- | ------------------------------------------------- |
> | Development | **acme-dev**      | The development environment and organization      |
> | Testing     | **acme-test**     | An environment for testing and production preview |
> | Production  | **acme-prod**     | The production environment and organization       |

Reviewer: ```suggestion
Depending on your Software Development Life Cycle (SDLC) you can create multiple organizations or projects to keep your applications environments seperated.
```
